docs/pages/about.tsx

```diff
@@ -22,34 +17,17 @@ export default function About(props: {
         <h1 className={styles.title}> About </h1>
         <Data text={props.data + ", this is just a test"} sn="LinkedIn" />
         <div className={styles.grid}>
-          {experience.map((e) => {
+          {studies.map((e) => {
             return (
-              <a className={styles.card} key={e.place}>
-                <h2>{e.place}</h2>
-                <h3>
-                  Studies:{" "}
-                  {e.studies.map((s) => {
-                    return s.title + "\n";
-                  })}
-                </h3>
-                <h3>
-                  Work:{" "}
-                  {e.works.map((w) => {
-                    return w.position + "\n";
-                  })}
-                </h3>
+              <a className={styles.card} key={e.title + "-" + e.topic}>
+                <h2>
+                  {e.topic}, {e.title}
+                </h2>
+                <h3>{e.place}</h3>
+                <p>{e.description}</p>
```

Comment:
```suggestion
     <Study title="B.S." topic="Computer Science" ...>
```

We may want to have a list of this?